server/modules/data-access/cachedSingleAccountDynamoTable.js

```diff
@@ -2,94 +2,12 @@
 
 'use strict';
 
-let mkArn = require('modules/data-access/dynamoTableArn').mkArn;
-let { attachAuditMetadata, updateAuditMetadata } = require('modules/data-access/dynamoAudit');
-let describeDynamoTable = require('modules/data-access/describeDynamoTable');
-let hashKeyAttributeName = require('modules/data-access/dynamoTableDescription').hashKeyAttributeName;
-let tableArn = require('modules/data-access/dynamoTableDescription').tableArn;
-let { makeWritable } = require('modules/data-access/dynamoItemFilter');
-let dynamoVersion = require('modules/data-access/dynamoVersion');
-let dynamoTable = require('modules/data-access/dynamoTable');
+let singleAccountdynamoTable = require('modules/data-access/singleAccountdynamoTable');
```

Comment:
Can you update this to be `require('modules/data-access/singleAccountDynamoTable')` (capital **D**) to avoid case-sensitivity issues on Linux?